guide/react-native/list-other.mdx

```diff
@@ -2,20 +2,20 @@
 sidebar_position: 4
 ---
 
-# List other participants
+# List other peers
 
-In order see other streaming participants, you can use `useParticipants`. It will return all
-other participants, together with track that they are streaming.
+In order see other streaming peers, you can use `usePeers`. It will return all
+other peers, together with track that they are streaming.
```

Comment:
**polish:** 
```suggestion
In order to see other streaming peers, you can use `usePeers`. It will return all
other peers, together with the tracks that they are streaming.
```